app/src/pages/inside/common/modals/editItemsModal/editItemsModal.jsx

```diff
@@ -330,6 +342,12 @@ export class EditItemsModal extends Component {
 
   showWarningMessage = () => this.setState({ warningMessageShown: true });
 
+  validateAttributes =
+    !validate.attributesArray(this.props.currentAttributes) && this.props.isSaveButtonPressed;
+
+  onAddAttribute = () =>
+    this.props.isSaveButtonPressed && this.props.setIsSaveButtonPressedAction(false);
```

Comment:
There is no need to handle form 'submission' and attributes validation in this case. You can use `invalid` prop form [redux-form](https://redux-form.com/8.3.0/docs/api/props.md/#-code-invalid-boolean-code-) to understand is the error exists or no and show appropriate warning message.